pyproject.toml

```
@@ -19,3 +75,24 @@ force-exclude = '''
 [tool.isort]
 profile = 'black'
 skip_glob = 'signac/_vendor/*'
+
+[tool.pydocstyle]
+convention = "numpy"
+match = "^((?!\\.sync-zenodo-metadata|setup).)*\\.py$"
+match-dir = "^((?!\\.|tests|_vendor).)*$"
+ignore-decorators = "deprecated"
+add-ignore = "D105, D107, D203, D204, D213"
+
+[tool.mypy]
+ignore_missing_imports = true
+
+[tool.pytest.ini_options]
+xfail_strict = true
+filterwarnings = "error"
+
+[tool.coverage.run]
+branch = true
+concurrency = [ "thread" , "multiprocessing" ]
```

Is all this whitespace a TOML requirement? I prefer formatting like:
```suggestion
concurrency = ["thread", "multiprocessing"]
```

---

No, it is not required.